Px.Utils/PxFile/Data/PxFileStreamDataReader.cs

```diff
@@ -10,14 +11,11 @@ public sealed class PxFileStreamDataReader : IPxFileStreamDataReader, IDisposabl
     {
         private readonly Stream _stream;
         private readonly PxFileSyntaxConf _conf;
-        private long _dataStart;
+        private long readIndex = 0;
 
         private const int INTERNAL_BUFFER_SIZE = 4096;
```

Comment:
Could this be parameterizable for example to help testing?

Reply:
Changed it according to the feedback